.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2017,
```

Comment:
Hmm, so from the TypeScript side, the syntax we are allowed to use is a function of both the TypeScript version we use, and the compiler options (like `lib`, which is currently set at es2015). How do these play together? Are we forced to specify ecmaVersion here?

I am wondering if we would be better off just introducing Prettier for formatting, and leaving static checks to TypeScript for the time being. I wonder how much overlap there is between correctness checks in ESLint and the TS compiler's responsibility.

Reply:
Discussed offline, there shouldn't be any interference.

Reply:
Im going to set the ecmaVersion here to 2015 to match the current tsconfig `lib`, as you noted here!